packages/auth/src/core/credentials/email.test.ts

```diff
@@ -107,7 +108,14 @@ describe('core/credentials/email', () => {
             recaptchaKey: 'site-key'
           });
 
-          auth.setRecaptchaConfig({emailPasswordEnabled: true});
+      mockEndpointWithParams(Endpoint.GET_RECAPTCHA_CONFIG, {
+        clientType: RecaptchaClientType.WEB,
```

Comment:
Indentation in this file is still weird

Reply:
Hmm, seems the linter doesn't work here. Fixed.